src/selects/channelSelect.ts

```diff
@@ -43,8 +48,18 @@ export async function execute(interaction: SelectMenuInteraction) {
             }
         }
     }
+
+    const groupId = interaction.customId.substring(
+        interaction.customId.lastIndexOf('-') + 1
+    );
+
+    const select = data.find((x) => x.id == groupId);
+    if (!select || !select.data) {
+        return;
+    }
+
     await interaction.update({
-        components: [...data],
+        components: [...select.data],
```

Comment:
Using only update leads to error if user adds too many channels at once maybe before for add `await interaction.deferUpdate();` and change `interaction.update({components: [...select.data], fetchReply: true});` to `    await interaction.editReply({ components: [...select.data] });`